spec/models/comment_spec.rb

```diff
@@ -0,0 +1,80 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  let(:test_movie) { Movie.new }
+  let(:test_user) { User.create!(name: "Carlos", email: "Carlos@gmail.com", password: "password" ) }
+
+  describe 'validations' do
+
+    before(:each) do
+      test_movie.get_movies_by_title("Fight Club")
+      first_movie = Movie.first
+      @new_comment = first_movie.comments.new(content: "This film was a work of art", user_id: test_user.id)
```

Comment:
No need for instance variables in your tests, since none of these variables are being passed around.